samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/KMLLayers.razor

```diff
@@ -0,0 +1,53 @@
+﻿@page "/kmllayers"
+
+<h3>Kml Layers</h3>
+
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-KMLLayer.html">ArcGIS API for JavaScript</a>
+</div>
```

Comment:
Add a second link for the KML Layer source (see other sample pages for example).

Reply:
source url is now included and reformatted the page to display better.